### PR TITLE
Improve Presets Tooltip

### PR DESCRIPTION
**Pull request by Maplesstar**
Added a brief description of the beginner and easy presets to hopefully alleviate some new player confusion.

### CODE REVIEW

**Comment by ETR-BTF** [comment]
I think the phrasing "timesaver skips" is a little confusing. I initially thought you were talking about logic tricks, but the only thing that makes sense is if you meant excluded locations. Maybe it's best to explicitly mention that neither preset has logical tricks enabled, and that the easy preset excludes some of the harder or more cumbersome checks. 

Is it also worth mentioning the other presets? There's only hell mode and a bunch of tournament presets that could be grouped together as "the other presets are just tournament presets". I think the aforementioned presets can be described without making the tooltip significantly bigger.

**Comment by Maplesstar** [comment]
I made this after that looong discussion in #general about the beginner and easy presets specifically for new players, which is why I didn't mention the other presets. By "timesaver skips" I meant the section on the Other tab named timesavers, again because that was the point of contention. 

The presets are alphabetized except for the hardcoded ones. That's why hell mode is where it is.

But good points, I'll try to update this tooltip to better cover everything if no one else comes up with better wording first.

File: SettingsList.py [hunk 1]
```diff
@@ -1870,11 +1870,19 @@ def __init__(self, name, gui_text, min, max, default, step=1,
...
-            and/or wish to dive right in. Most timesaver skips are enabled and the world is 
-            more open after leaving Kokiri Forest.
+            Easy Mode is aimed and those who have perhaps seen a few randomizer runs previously and/or 
+            wish to dive right in. Uses base glitchless logic. Most timesavers (See the tab "Other") 
+            are enabled and the world is more open after leaving Kokiri Forest.
+
+            Hell Mode enables every setting to provide maximum randomness, but still uses glitchless
+            logic to ensure a beatable seed. However, be aware that all glitchless "tricks" are enabled
+            which have the potential to require the player to perform difficult techniques. 
+            Expect an long playthrough, even with good note-taking.
```
Comment (ETR-BTF) [inline SettingsList.py:1883]: an -> a

I'm very sorry about this lmao

**Review by cjohnson57 — APPROVED** [review]
Seems good to me, think this will be helpful to have

**Review by Cuphat — APPROVED** [review]

**Pull request merged**